lib/system/Contexts.hpp

```diff
@@ -88,6 +97,7 @@ namespace MAT_NS_BEGIN {
         unsigned                             maxUploadSize = 0;
         EventLatency                         latency = EventLatency_Unspecified;
         std::map<std::string, size_t>        packageIds;
+        std::string                          traceId;
```

Comment:
nit - this can be behind macro too.

Reply:
sure. will take care while merging in 1DS main repo